components/GitHubEventAction.tsx

```diff
@@ -61,7 +77,24 @@ function GitHubEventAction({type, payload}: GitHubEventActionProps): JSX.Element
       if (!action || !prNum || !prURL) {
         return unknown;
       }
-      return <span>{action} <ELink link={prURL}>pull request #{prNum}</ELink> in</span>;
+      return (
+        <span>
+          {action} <ELink link={prURL}>pull request #{prNum}</ELink> in
+        </span>
+      );
+    }
+    case 'PullRequestReviewComment': {
+      const action = payload.action;
+      const author = payload.comment.user.login;
+      const authorLink = payload.comment.user.url;
+      const prNum = payload.comment.number;
+      const prLink = payload.comment.pull_request_url;
+      return (
+        <span>
+          {action} by <ELink link={authorLink}>{author}</ELink> on{' '}
+          <ELink link={prLink}> pull request #{prNum}</ELink> in
+        </span>
+      );
```

Comment:
You shouldn't need to specify the author; currently, the component that wraps `<GitHubEventAction />` (`<GitHubEvent />`) handles this logic!

(that means you shouldn't need the `author` or `authorLink` fields).

Reply:
yepyep